src/components/ItemList.svelte

```diff
@@ -155,7 +165,7 @@
       </h3>
       {#if totalItems > 0}
         <p>
-          {totalItems} expired or removed {itemType} found.
+          {totalItems} expired, removed or hidden {itemType} found.
```

Comment:
let's also add this to line 318 of this file

Reply:
Thanks, done!